sklearn/linear_model/_quantile.py

```diff
@@ -47,15 +47,8 @@ class QuantileRegressor(LinearModel, RegressorMixin, BaseEstimator):
         Method used by :func:`scipy.optimize.linprog` to solve the linear
         programming formulation.
 
-        From `scipy>=1.6.0`, it is recommended to use the highs methods because
-        they are the fastest ones. Solvers "highs-ds", "highs-ipm" and "highs"
-        support sparse input data and, in fact, always convert to sparse csc.
```

Comment:
I think it's worth keeping this docstring, but remove the `scipy>=1.6.0` part.

Reply:
reverted this change.